### PR TITLE
Add resize event and disable pointer events on iframes on trigger of resize

### LINKED ISSUE #3986: Drag into Iframe doesn't see mouseup event
### Replicate
1. Create and/or navigate to a Web Page view, which uses an Iframe.
2. Click and drag to resize and expand the tree view. Observe that as you drag to the right into the Iframe's area, the tree pane will stop resizing. 
3. Release the mouse to "stop" dragging, and move your cursor back into the tree area; as you do so, observe that the tree will shrink as you move the cursor (as though the mouse is still down and dragging), even though the mouse button is no longer being held down. 

The same phenomena occurs when expanding the size of the Inspector pane.

**Expected behavior**: the user should be able to resize the pane via dragging as normally and as seen in other view types. Marking as critical as this is manifesting while utilizing newly integrated components that utilize Iframes.

### PR DESCRIPTION
**Pull request by khalidadil**
### Resolves

#3986

### Author Checklist

Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | N/A
Command line build passes? | Y
Changes have been smoke-tested? | Y
Testing instructions included? | N/A

Testing instructions are included in #3986

### CODE REVIEW

**Comment by davetsay** [comment]
@charlesh88 , the approach and the scss change looks good to me. Can you give @khalidadil your stylistic blessing?

File: src/ui/layout/Layout.vue [hunk 1]
```diff
@@ -45,14 +45,16 @@
     <div class="l-shell__drawer c-drawer c-drawer--push c-drawer--align-top"></div>
 
     <multipane
-        class="l-shell__main"
+        :class="['l-shell__main', isResizing ? 'l-shell__resizing' : '']"
```
Comment (davetsay) [inline src/ui/layout/Layout.vue:48]: This is fine and you'll see this a lot in our codebase. We should try to bind dynamic styling to computed class and style objects though so these don't get out of hand. Static classes can be left in their own non bound attribute

**Review by davetsay — CHANGES_REQUESTED** [review]
This looks good. I have a couple of minor code style changes.

Can we change `openmct-start-drag` and `openmct-end-drag` to `start-resizing` and `end-resizing` to be consistent with the rest of the feature flow?

One comment inline about vue class bindings.

**Comment by unlikelyzero** [comment]
@khalidadil there is current a failing test on master for `ci/circleci: node10-chrome` and I'm investigating now

**Review by davetsay — APPROVED** [review]
LGTM

## Reviewer Checklist
| | |
| --- | :---: |
| Changes appear to address issue? | Y |
| Appropriate unit tests included? | N/A |
| Code style and in-line documentation are appropriate? | Y |
| Commit messages meet standards? | Y |
| Has associated issue been labelled `unverified`? | Y |

**Comment by unlikelyzero** [comment]
@khalidadil I updated your branch with the master test fixes and it's gone through

**Pull request merged**